recipes/raylib/all/conanfile.py

```diff
@@ -17,13 +18,19 @@ class RaylibConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "build_examples": [True, False],
+        "use_external_glfw": [True, False],
```

Comment:
```suggestion
        "use_external_glfw": [True, False],
```

Why not fix glfw package to support Android instead?

Reply:
From https://www.glfw.org/ 
"GLFW is written in C and supports Windows, macOS, X11 and Wayland."

Reply:
I assume Android builds need to use some `glfw` port provided in the Android NDK bundle, right?

If that is the case I would simplify the recipe and remove `use_external_glfw` option. The regular build will use always requirements from ConanCenter, and the Android build (probably others like Emscripten) will use always the bundled library (like you do with `opengl/system`).

Does it make sense?

In the future, if someone implements `glfw` library for Android, we can use it here as well.